package.json

```diff
@@ -118,7 +118,7 @@
     "react-native-blob-util": "0.15.0",
     "react-native-calendar-events": "2.2.0",
     "react-native-camera": "^3.44.3",
-    "react-native-config": "1.4.3",
+    "react-native-config": "^1.4.5",
```

Comment:
Having `react-native-config` at `1.4.3` is going to conflict with the new Android build settings required by `react-native-vision-camera`, so yes it is required to have this feature. Do you think it could cause problems?

Reply:
No I don't. Further tests will tell us if there will be some issues